blacklight-spotlight.gemspec

```diff
@@ -43,23 +43,27 @@ these collections.)
   s.add_dependency 'iiif-presentation'
   s.add_dependency 'leaflet-rails'
   s.add_dependency 'legato'
+  s.add_dependency 'mimemagic', '0.3.8'
   s.add_dependency 'mini_magick'
   s.add_dependency 'nokogiri'
   s.add_dependency 'oauth2'
   s.add_dependency 'openseadragon'
   s.add_dependency 'ostruct', '!= 0.3.0', '!= 0.3.1', '!= 0.3.2'
   s.add_dependency 'paper_trail', '>= 11.0', '< 13'
+  s.add_dependency 'pg'
```

Comment:
I think it is inappropriate to make `pg` a hard dependency of spotlight. It works with other databases too (same for sidekiq, webpacker, etc)